src/simulation/m_monopole.fpp

```diff
@@ -16,7 +16,7 @@ module m_monopole
 
```

Comment:
rename from `m_monopole` to `m_acoustic_src`, perhaps?

Reply:
Yes! This is one of the next steps. 

Wanted to ask you if I should also change the `Monopole` option in case file to `acoustic_src` or `acoustic_source`, so we can change everything in the documentation together? This would be much cleaner but it also means changing all existing case files containing `Monopole` though.

Reply:
@sbryngelson Would you like me to perform this change? Thanks!

Reply:
i think `acoustic_source` is preferable, even though it is a 'breaking change' of sorts. a quick find/replace on the examples and toolchain directories should take care of most of the changes.

Reply:
That's what I think too! Will change all relevant places